playbooks/deprovision.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: "Play to deprovision argo"
+
+  hosts: localhost
+  connection: local
+
+  # Only if not provided
+  vars_prompt:
+    - name: namespace
+      prompt: "Namespace"
+      private: no
+
+  tasks:
+  - name: "delete argo workflow resources"
+    tags:
+      - argo
+      - argo-workflows
+    command: "oc delete -n {{ namespace }} {{ item }} --all"
```

Comment:
`delete --all` seems to be dangerous, cant we select them by some `app=argo` label?

Reply:
Yep, correct, good point... I was deprovisioning a names pace dedicated just to Argo so it wasn't a big deal, but ye, I will fix it.

Reply:
This commit introduces the functionality https://github.com/CermakM/ansible-role-argo-workflows/commit/bce58474b2e1ea46514bf02698acb43d8e199aa8